nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -86,32 +86,21 @@ def target_point(target_type: TargetType, target_node_name: str, port_id: int) -
         return ONNXTargetPoint(target_type, target_node_name, port_id)
 
     @staticmethod
-    def create_activation_quantizer_insertion_command(
+    def create_quantizer_insertion_command(
         nncf_graph: NNCFGraph,
         target_point: ONNXTargetPoint,
         quantizer_config: QuantizerConfig,
         parameters: FakeQuantizeParameters,
-    ) -> ONNXQuantizerInsertionCommand:
-        nncf_input_node_next_nodes = ONNXMinMaxAlgoBackend._get_input_edges_mapping(nncf_graph)
-        axis = ONNXMinMaxAlgoBackend._get_axis(nncf_graph, target_point, quantizer_config)
-        tensor_type = np.int8 if np.any(parameters.input_low < 0) else np.uint8
-        onnx_parameters = convert_fq_params_to_onnx_params(parameters, quantizer_config.num_bits, tensor_type, axis)
-        return ONNXQuantizerInsertionCommand(target_point, nncf_input_node_next_nodes, onnx_parameters)
-
-    @staticmethod
-    def create_weight_quantizer_insertion_command(
-        nncf_graph: NNCFGraph,
-        target_point: ONNXTargetPoint,
-        quantizer_config: QuantizerConfig,
-        parameters: FakeQuantizeParameters,
-    ) -> ONNXQuantizerInsertionCommand:
-        if quantizer_config.signedness_to_force is False:
-            raise ValueError(
-                "The HW expects to have signed quantization of weights, "
-                "while the quantizer configuration for weights contains signedness_to_force=False."
-            )
-
-        tensor_type = np.int8  # The weight is restricted to have only signed range
+    ):
+        if target_point.is_weight_target_point():
+            tensor_type = np.int8  # The weight is restricted to have only signed range
+            if quantizer_config.signedness_to_force is False:
```

Comment:
Is it possible condition? Looks like that it should be checked on creating quantizer_config

Reply:
Removed this check as it is already done inside `assign_qconfig_lists_to_modules`